force-app/main/default/lwc/donationHistoryTable/donationHistoryTable.js-meta.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<LightningComponentBundle xmlns="http://soap.sforce.com/2006/04/metadata">
+    <apiVersion>52.0</apiVersion>
+    <isExposed>true</isExposed>
```

Comment:
```suggestion
    <isExposed>false</isExposed>
```

I don't think this component needs to be exposed, we're not expecting our end-users to create their own components from this component.